godo.go

```diff
@@ -156,6 +157,17 @@ func addOptions(s string, opt interface{}) (string, error) {
 	return origURL.String(), nil
 }
 
+// NewFromToken returns a new DigitalOcean API client with the given API
+// token.
+func NewFromToken(token string) *Client {
+	ctx := context.Background()
+
+	config := &oauth2.Config{}
```

Comment:
nit: there seems to be a mix of this vs `new(oauth2.Config)` syntax throughout the code base. Do we have a preferred method?

Reply:
I prefer either `var config *oauth2.Config` or `config := &oauth2.Config{}`.

Using `new(oauth2.Config)` make it less obvious (in my opinion) that the returned type has a pointer receiver, compared to the first two examples where you can see the `*` or `&` which lets you know your variable has a pointer receiver.